py/tests/set.py

```diff
@@ -55,6 +55,17 @@
 
 d = a ^ b
 assert 1 in c
+
+doc="set"
+a = set([1,2,3])
+b = set("set")
+c = set((4,5))
+assert 1 in a
+assert 2 in a
+assert 3 in a
```

Comment:
@SanggiHong 
Would you like to add below codes for the test? 
```python
assert len(a) == 3
assert len(b) == 3
assert len(c) == 2
```

Reply:
No problem, just added :)